code/modules/projectiles/firing.dm

```diff
@@ -81,8 +81,8 @@
 
 			//Split Y+Pixel_Y up into list(Y, Pixel_Y)
 			var/list/screen_loc_Y = splittext(screen_loc_params[2],":")
-			var/x = text2num(screen_loc_X[1]) * 32 + text2num(screen_loc_X[2]) - 32
-			var/y = text2num(screen_loc_Y[1]) * 32 + text2num(screen_loc_Y[2]) - 32
+			var/x = text2num(screen_loc_X[1]) * world.icon_size + text2num(screen_loc_X[2]) - world.icon_size + (user.client ? user.client.pixel_x : 0)
+			var/y = text2num(screen_loc_Y[1]) * world.icon_size + text2num(screen_loc_Y[2]) - world.icon_size + (user.client ? user.client.pixel_y : 0)
```

Comment:
I don't know why client.pixel_x and client.pixel_y need to be added here to adjust for sniper accuracy. On TG, this isn't the case and they're perfectly accurate without it.